pycarwings2/pycarwings2.py

```diff
@@ -118,7 +118,12 @@ def _request(self, endpoint, params):
         else:
             params["custom_sessionid"] = ""
 
```

Comment:
the change here adds `headers` as a dictionary, and this conforms to the documentation
https://docs.python.org/3/library/urllib.request.html#urllib.request.Request
:heavy_check_mark: